src/app/backups/hooks/useBackupDeviceActions.ts

```diff
@@ -1,7 +1,6 @@
-import { DriveFolderData } from '@internxt/sdk/dist/drive/storage/types';
 import { Device } from 'app/backups/types';
 import { deleteBackupDeviceAsFolder } from 'app/drive/services/folder.service';
-import { DriveItemData, DriveFolderData as DriveWebFolderData } from 'app/drive/types';
+import { DriveFolderData, DriveItemData } from 'app/drive/types';
```

Comment:
have you tried to update the sdk types in order to update them and use it in all projects?